test/basic.js

```diff
@@ -37,6 +37,42 @@ test('connect and connack (minimal)', function (t) {
   })
 })
 
+test('connect and connackSent event', function (t) {
+  var s = setup()
+  var clientId = 'my-client'
+
+  t.plan(2)
+  t.timeoutAfter(50)
+
+  s.broker.on('connackSent', function (client) {
+    t.equal(client.id, clientId, 'connackSent event')
+  })
+
+  s.inStream.write({
+    cmd: 'connect',
+    protocolId: 'MQTT',
+    protocolVersion: 4,
+    clean: true,
+    clientId: clientId,
+    keepalive: 0
+  })
+
+  s.outStream.on('data', function (packet) {
+    t.deepEqual(packet, {
+      cmd: 'connack',
+      returnCode: 0,
+      length: 2,
+      qos: 0,
+      retain: false,
+      dup: false,
+      topic: null,
+      payload: null,
+      sessionPresent: false
+    }, 'successful connack')
+    t.end()
+  })
+})
```

Comment:
Can you please move this at the end of https://github.com/mcollina/aedes/blob/master/test/meta.js?